dev-docs/modules/userId.md

```diff
@@ -69,6 +69,54 @@ of sub-objects. The table below has the options that are common across ID system
 
 ## User ID Sub-Modules
 
+### AdmixerID
+
+Admixer ID, provided by [Admixer](https://admixer.com/), is a universal ID solution that doesn’t rely on 3rd party cookies and gives publishers access to a wide range of Admixer’s demand sources, amplifying audience segmentation, targeting and measurement.
+This sub-module enables the Admixer ID to be available in the bid request.
```

Comment:
All UserId module sub adapters propagate their ID into the bid stream, explicitly calling this out may cause publishers to doubt other sub adapters.

Reply:
line removed.